fx.go

```diff
@@ -11,9 +15,12 @@ import (
 	"github.com/urfave/cli"
 )
 
+const Version = "0.7.3"
```

Comment:
Since we're not pass `version` from build command, so I'm thinking it'd better naming the variable in lower case, otherwise, the linter will require to have comments above it.